server/planning/types/delivery.py

```diff
@@ -0,0 +1,18 @@
+from pydantic import Field
+from typing import Annotated
+from datetime import datetime
+
+from superdesk.core.resources import fields
+from superdesk.core.resources.validators import validate_data_relation_async
+from .base import BasePlanningModel
+
+
+class DeliveryResourceModel(BasePlanningModel):
```

Comment:
The `BasePlanningModel` adds `original_creator` and `version_creator` to the model. Delivery resources are managed internally, and therefor do not have any user creator information (see the original schema).

Because of this, you could use core resource & service directly instead.
i.e.

```python
# file: planning/types/delivery.py
from superdesk.core.resources import ResourceModelWithObjectId

class DeliveryResourceModel(ResourceModelWithObjectId):
    ...

# file: planning/assignments/delivery_service.py
from superdesk.core.resources.service import AsyncResourceService
from planning.types import DeliveryResourceModel

class DeliveryAsyncService(AsyncResourceService[DeliveryResourceModel]):
    ...
```